sis_scraper/main.py

```diff
@@ -427,7 +427,21 @@ def optimize_column_ordering(data, num_columns=3):
                                         ] = 1
 
                     conflicts[section["crn"]] = "".join(str(e) for e in conflict)
-
+        # If 0 courses, or exactly one course, occupies a slot in the conflict bitstring, we can safely remove it to save space
+        # as that implies no conflicts are possible in that block.
+        # The following code computes a list of candidates that fit this criteria
+        prune = [bit_index for bit_index in range(0, BIT_VEC_SIZE)
```

Comment:
I'd like it if this variable name was changed to something a little more descriptive... "unnecessary_indices", maybe?

Reply:
Applied in latest change (force push amend)